packages/playwright/src/index.ts

```diff
@@ -185,8 +185,16 @@ export default class AxeBuilder {
 
   private async inject(frames: Frame[]): Promise<void> {
     for (const iframe of frames) {
-      await iframe.evaluate(await this.script());
-      await iframe.evaluate(await this.axeConfigure());
+      const race = new Promise((_, reject) => {
+        setTimeout(() => {
+          reject(new Error('Script Timeout'));
+        }, 1000);
+      });
+      const evaluate = iframe.evaluate(this.script()).then(async () => {
```

Comment:
In the other cases (either racing a timeout or using a page load timeout), the 1000ms time races with a single evaluate invocation; here, it's racing two, so it's more likely to fail spuriously on a slow test machine. It only needs to race with the first evaluate call to do its job and detect an unloaded frame, so I think it'd be better to limit what it's racing with.